packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -31,14 +31,21 @@ const humanTimeDelta = timeMs => {
 }
 
 /* Create a cancellable promise for replication with default options */
-const startReplication = (pouch, getReplicationURL) => {
+const startReplication = (
+  pouch,
+  replicateFromRemoteOnly,
```

Comment:
I'm not a big fan of rewording the concept. One time we use : readOnly, and other ReplicateFromRemoteOnly. Can't we use one for both?

Reply:
Fair, i've renamed the variables to `ReplicateFromRemoteOnly`. However for the public API i'd rather keep `readOnly`/`readWrite` since I think `ReplicateFromRemoteOnly`/Replicate...BothWays?`is too wordy and the distinction between bidirectional and the others is hard to express.

Reply:
👍